src/items/itemsTable.component.tsx

```diff
@@ -147,6 +151,10 @@ export function ItemsTable(props: ItemTableProps) {
     'create' | 'duplicate' | 'edit'
   >('create');
 
+  const [openDialogAsPrivilegedUser, setOpenDialogAsPrivilegedUser] =
+    React.useState<boolean>(false);
+
+  // Breadcrumbs + Mui table V2 + extra
```

Comment:
```suggestion
  // Breadcrumbs + Mui table V2 + extra
```
I think this got added back in, from a merge. Can you remove it